lib/metadataTypes/definitions/TriggeredSendSummary.definition.js

```diff
@@ -0,0 +1,213 @@
+export default {
+    bodyIteratorField: 'Results',
+    dependencies: ['triggeredSend', 'folder'],
+    dependencyGraph: {
+        triggeredSend: ['r__triggeredSend_key'],
+    },
+    filter: {
+        r__folder_Path: [
+            'Journey Builder Sends/',
+            'HiddenCategory/HiddenJourneyBuilderTriggeredSends/',
+        ],
+    },
+    folderType: 'triggered_send_summary',
+    hasExtended: false,
+    idField: 'ObjectID',
+    keepId: true,
+    keyIsFixed: true,
+    keyField: 'CustomerKey',
+    nameField: 'Name',
+    folderIdField: 'TriggeredSend.CategoryID',
+    createdDateField: null,
+    createdNameField: null,
+    lastmodDateField: null,
+    lastmodNameField: null,
+    restPagination: null,
+    maxKeyLength:
+        null /** Key can't be changed, it's the same as the TriggeredSend key, metadata of this type can't be created either */,
```

Comment:
```suggestion
    maxKeyLength: 36,
```

copied from triggeredSend.definition